docs/integrations/aws_lambda.md

```diff
@@ -0,0 +1,93 @@
+3. Use the CPython implementation.
+4. The Python version must be compatible with the Lambda runtime.
+5. This flag ensures that the package is installed pre-built binary wheels.
+6. The latest version of `pydantic` will be installed.
+
+## Troubleshooting
+
+!!! warning "How to fix the `no module named 'pydantic_core._pydantic_core'` error"
+    The
+    ```
+    no module named `pydantic_core._pydantic_core`
+    ```
+
+    error is a common issue that indicates you have installed `pydantic` incorrectly. To debug this issue, you can try the following steps (before the failing import):
+
+    1. Check the contents of the installed `pydantic-core` package. Are the compiled library and its type stubs both present?
+
+    ```py test="skip" lint="skip"
+    from importlib.metadata import files
+    print([file for file in files('pydantic-core') if file.name.startswith('_pydantic_core')])
+    """
+    [PackagePath('pydantic_core/_pydantic_core.pyi'), PackagePath('pydantic_core/_pydantic_core.cpython-312-x86_64-linux-gnu.so')]
+    """
+    ```
+
+    You should expect to see two files like those printed above. The compile library file will be a .so or .pyd with a name that varies according to the OS and Python version.
+
+    2. Check that your lambda's Python version is compatible with the compiled library version found above.
+
+    ```py test="skip" lint="skip"
+    import sysconfig
+    print(sysconfig.get_config_var("EXT_SUFFIX"))
+    #> '.cpython-312-x86_64-linux-gnu.so'
+    ```
+
+    You should expect to see the same suffix here as the compiled library, for example here we see this suffix `.cpython-312-x86_64-linux-gnu.so` indeed matches `_pydantic_core.cpython-312-x86_64-linux-gnu.so`.
+
+    If these two checks do not match, your build steps have not installed the correct native code for your lambda's target platform. You should adjust your build steps to change the version of the installed library which gets installed.
+
+    Most likely errors:
+
+    * Your OS or CPU architecture is mismatched (e.g. darwin vs x86_64-linux-gnu). Try passing correct `--platform` argument to `pip install` when installing your lambda dependencies, or build inside a linux docker container for the correct platform. Possible platforms at the moment include `--platform manylinux2014_x86_64` or `--platform manylinux2014_aarch64`, but these may change with a future Pydantic major release.
+
+    * Your Python version is mismatched (e.g. `cpython-310` vs `cpython-312`). Try passing correct `--python-version` argument to `pip install`, or otherwise change the Python version used on your build.
+
+
+## Extra Resources
+
+### More Debugging Tips
+
+If you're still struggling with installing `pydantic` for your AWS Lambda, you might consult with [this issue](https://github.com/pydantic/pydantic/issues/6557), which covers a variety of problems and solutions encountered by other developers.
+
+
+### Blog Posts
```

Comment:
```suggestion
### Validating `event` and `context`
```
(or similar).

Functionally this is the same, but I think it seems marginally more likely to usefully attract reader attention — I doubt a user will be like "oh blog posts — that's what I need", vs. more likely to react with "oh validating the inputs to a lambda — that's what I need".

Even if we do eventually add more blog posts, I still think it makes sense to use a category heading based on the content rather than the type of item.